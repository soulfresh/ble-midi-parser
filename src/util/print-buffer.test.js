/* eslint-env jest */
import printBuffer from './print-buffer';

test('printBuffer(buffer)', () => {
  const buffer = new Buffer([
    0,
    1,
    2,
    3,
    4,
    5,
    6,
    7,
    8,
    9,
    10,
    11,
    12,
    13,
    0,
    1,
    126,
    127,
    128,
    129,
    130,
    253,
    254,
    255,
    256,
    257,
    258,
    259,
    260,
    261,
    262
  ]);
  expect(printBuffer(buffer)).toMatchSnapshot();
});
