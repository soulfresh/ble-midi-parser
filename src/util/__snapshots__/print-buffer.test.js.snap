// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`printBuffer(buffer) 1`] = `
"00000000
00000001
00000010
00000011
00000100
00000101
00000110
00000111
00001000
00001001
00001010
00001011
00001100
00001101
00000000
00000001
01111110
01111111
10000000
10000001
10000010
11111101
11111110
11111111
00000000
00000001
00000010
00000011
00000100
00000101
00000110"
`;
